<?xml version="1.0"?>
<launch>
    <!-- Start the usb webcam -->
    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen" respawn="true">
        <param name="video_device" type="string" value="/dev/video1" />
        <param name="pixel_format" value="yuyv" />
        <param name="image_width"  value="640" /> 
        <param name="image_height" value="480" />
    </node>

    <rosparam command="load" file="$(find youbot_description)/controller/arm_joint_universal_control.yaml"/>
    <rosparam command="load" file="$(find raw_hardware_config)/$(env ROBOT)/config/arm.yaml"/>
    
    <node pkg="raw_visual_servoing" type="visual_servoing_node" name="c" ns="raw_visual_servoing" respawn="true" launch-prefix="gdb -ex run --args" output="screen">
        <param name="operating_mode" type="int" value="1" />
    </node>
</launch>